csds-client/client/client.go

```diff
@@ -0,0 +1,188 @@
+package client
+
+import (
+	"context"
```

Comment:
Generally nicer to split named and unnamed imports into two blocks separated by a new line.

Reply:
Solved.

Reply:
Run goimports? https://github.com/golang/tools/commit/ed69e84b1518b5857a9f4e01d1f9cefdcc45246e